release-notes/major11/11.3.0/valtimo-frontend-libraries.md

```diff
@@ -2,11 +2,19 @@
 
 ## New Features
 
-No new features.
+The following features were added:
+
+* **Ability to use value resolvers to set date in `Create Zaakbesluit` plug-in action configuration**
+    
+    It is now possible to use process variable or document json path to set `Starting date` and `Expiration date` in `Create Zaakbesluit` plug-in action configuration so that each besluit has its own start and expiration date.
 
 ## Bugfixes
 
-No bugfixes.
+The following bugs were fixed:
+
+* **Plug-in action configuration can be completed without filling mandatory field**
+
+  `Create Zaakbesluit` plug-in action configuration can be completed without filling mandatory `Starting date` field. It is now fixed.
```

Comment:
```suggestion
  `Create Zaakbesluit` plug-in action configuration can be completed without filling mandatory `Starting date` field.
```

This is in the bugfixes section, so if a bug is mentioned here, it only makes sense it's fixed. So there's no need to write that part down.